src/examples/select_partial_dataset_cpp17.cpp

```diff
@@ -0,0 +1,97 @@
+/*
+ *  Copyright (c), 2017, Adrien Devresse
+ *
+ *  Distributed under the Boost Software License, Version 1.0.
+ *    (See accompanying file LICENSE_1_0.txt or copy at
+ *          http://www.boost.org/LICENSE_1_0.txt)
+ *
+ */
```

Comment:
I don't mind the author to be Adrien, or who ever can compose the CMake file for me. Help wanted here to compose the `CMakeLists.txt`.